files/50unattended-upgrades

```diff
@@ -1,4 +1,3 @@
-/etc/apt/apt.conf.d/50unattended-upgrades 
 // Automatically upgrade packages from these (origin:archive) pairs
 Unattended-Upgrade::Allowed-Origins {
 	"${distro_id}:${distro_codename}-security";
```

Comment:
No, ideally all updates would be rolled out as part of an ansible solution. However, auto updating "updates", "backports" and "proposed" has a high likelihood of breaking inter-dependent packages. Also the pentest only requested that security auto updates be enabled.